Tangem/Modules/LegacyTokenList/LegacyTokenListView.swift

```diff
@@ -28,6 +28,8 @@ struct LegacyTokenListView: View {
             AlertToast(type: .complete(Colors.Icon.accent), title: Localization.contractAddressCopiedMessage)
         }
         .searchableCompat(text: $viewModel.enteredSearchText.value)
+        .keyboardType(.alphabet)
+        .disableAutocorrection(true)
```

Comment:
NP: Можно и без `true` даже ))

```suggestion
        .disableAutocorrection()
```

Reply:
> NP: Можно и без `true` даже ))

Да окей, смотрел что в остальных местах пишем, обновлю